repository/mysql/operator/templates/mysql.yaml

```diff
@@ -49,9 +52,9 @@ spec:
           value: kudo
         ports:
         - containerPort: 3306
```

Comment:
should {{ .Params.PORT }} replace 3306 ?

Reply:
Good spot, although taking another look I've realised that having the port as a parameter doesn't make sense given how it's currently deployed.  I've removed it for now.

Reply:
LGTM